server/jetstream_events.go

```diff
@@ -210,5 +210,8 @@ const JSServerOutOfStorageAdvisoryType = "io.nats.jetstream.advisory.v1.server_o
 // JSServerOutOfSpaceAdvisory indicates that a stream has lost quorum and is stalled.
 type JSServerOutOfSpaceAdvisory struct {
 	TypedEvent
-	Server string `json:"server"`
+	Server   string `json:"server"`
+	ServerID string `json:"server_id"`
+	Stream   string `json:"stream,omitempty"`
+	Cluster  string `json:"cluster"`
```

Comment:
basically just adding enough here so someone can really super easily figure out exactly what server it is.

Since we know which stream caused the failure I figured we might as well add that when possible, this is if people do weird mounting things as per recent calls.